<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FacadeClipper</class>
 <widget class="QMainWindow" name="FacadeClipper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FacadeClipper</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListView" name="listView"/>
      </item>
      <item>
       <widget class="QComboBox" name="methodsList">
        <item>
         <property name="text">
          <string>Semi Automatic Clipper</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Zones extractor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Same texture search engine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Power Watershed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Perspective Correction</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FloodFill</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Canny</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hough</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mainApplyButton">
        <property name="text">
         <string>Apply</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mainResetButton">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>667</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>&amp;Menu</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionOuvrir">
   <property name="text">
    <string>&amp;Ouvrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
